src/test/unit/model/indexing/lvalue_test.cpp

```diff
@@ -469,7 +472,9 @@ TEST(ModelIndexing, lvalueMatrixUniMulti) {
   test_throw(x, index_list(index_uni(0), index_min_max(2, 4)), y);
   test_throw(x, index_list(index_uni(5), index_min_max(2, 4)), y);
   test_throw(x, index_list(index_uni(2), index_min_max(0, 2)), y);
-  test_throw_ia(x, index_list(index_uni(2), index_min_max(2, 5)), y);
+  test_throw(x, index_list(index_uni(2), index_min_max(2, 5)), y);
```

Comment:
I assume `index_uni` means a single index value? What about `index_min_max`?

Reply:
`index_min_max` for vectors is a slice from `min:max` and for matrices a slice of rows from `min:max` (or columns from `min:max` if it's the second indexing argument)